doc/rst/source/cookbook/include-figures.rst

```diff
@@ -63,16 +63,17 @@ accomplished.
    Corel Draw will allow you to edit the BoundingBox graphically.
 
 -  A command-line alternative is to use freely-available program
-   **epstool** from the makers of Aladdin ghostscript. Running
+   *epstool* from the makers of `Aladdin ghostscript <https://www.ghostscript.com/>`_.
```

Comment:
@PaulWessel, can you provide some guidance as the appropriate way to refer to the owner of these tools? From the wikipedia page, it seems Aladdin was forked and then merged again before being bought by Artifex. I cannot find any current web presence for Aladdin Enterprise.

I would actually prefer that we only document the GMT way to do these tasks, so that we don't need to maintain the lists of other tools. An example of the downside for listing alternative programs is that Aldus Freehand is listed as an option to edit the BoundingBox, which had it’s final release in 2003.

Reply:
That section is ancient and reminisces of a simpler time...not.  Yes, I agree we should prune things like eps tool etc.  Our way with **psconvert** is superior given our settings etc.

Reply:
Sounds good, I can submit an update to that section as a separate PR afterwards.